rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -836,10 +835,10 @@ public void releaseBtc(Transaction rskTx) throws IOException {
         Address btcDestinationAddress = BridgeUtils.recoverBtcAddressFromEthTransaction(rskTx, btcParams);
         logger.debug("[releaseBtc] BTC destination address: {}", btcDestinationAddress);
 
-        requestRelease(btcDestinationAddress, pegoutValueInSatoshis, rskTx);
+        requestRelease(btcDestinationAddress, pegoutValueInWeis, rskTx);
     }
 
-    private void refundAndEmitRejectEvent(Coin value, RskAddress senderAddress, RejectedPegoutReason reason) {
+    private void refundAndEmitRejectEvent(co.rsk.core.Coin value, RskAddress senderAddress, RejectedPegoutReason reason) {
```

Comment:
I think this parameter can have a more meaningful name. Like `valueInWeis`, `refundValueInWeis`, or something. Wdyt?

Reply:
Updated